examples/hello_world/main.tf

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Google LLC. This software is provided as-is,
+# without warranty or representation for any use or purpose.
+# Your use of it is subject to your agreements with Google.
+
+provider "google" {
+  version = "~> 1.8"
```

Comment:
Small nitpick: Let's bump this to 1.19 or 1.20 or so.

Reply:
Bumped to `~> 1.20` and switched to examples/simple_example/main.tf following the cookiecutter.